### ISSUE TITLE
[Sign up] Not allowed to sign up with a duplicate email, fresh id number, (and vice versa) but there was no error

### ISSUE THREAD

**Issue by claudesedillo** [AlexAgnoII/SOFENGG#29]
![duplicate email](https://user-images.githubusercontent.com/25079774/33557380-0b727c88-d942-11e7-9afa-29dc88edc5ba.gif)
Tested By: Claude

Date Tested: 12-04-2017 10:08PM

Test Case Description:  Signing up with an e-mail that already exists in the database but with a fresh id number

Steps to reproduce error

- Sign up with an email that already exists in the database but with a new id number 
- For more detail, on the sign up page, the user must enter the following inputs:
- ID Number: 11436695 (Does not exist in DB)
- E-mail: claudesedillo@gmail.com
- Fill the rest of the fields with valid inputs
- Press sign up

Expected Output:
- The user should not be allowed to sign up because the email is already taken. An error message should pop up

Actual Output:
![email taken](https://user-images.githubusercontent.com/25079774/33557338-edf44b1e-d941-11e7-9af7-1eb377fab085.PNG)

- The log said that the email is already taken and the e-mail was not sent, although there was no error message. The prompt for validating the email was also shown

UPDATE: I also signed up with a fresh e-mail and a duplicate ID Number and the same output was shown. There was no error but it was displayed in the logs that the e-mail is new and the ID Number is a duplicate

TLDR: Wherever in the code checks if the email or id number exists in the DB, its checking ONLY IF both already exist, but not if only one exists

**Comment by ericembestro** [comment]
fixed necessary things for this issue. Please re-evaluate. Thanks

**Comment by claudesedillo** [comment]
![new sign up error](https://user-images.githubusercontent.com/25079774/33994009-dc6f66ce-e113-11e7-86b4-cb4d6f6d41ca.gif)

**Comment by ghost** [comment]
Fixed
